pkg/process/checks/system_info_windows.go

```diff
@@ -0,0 +1,64 @@
+package checks
+
+import (
+	"fmt"
+	"github.com/DataDog/gohai/cpu"
+	"github.com/DataDog/gohai/platform"
+	"strconv"
+
+	"github.com/DataDog/datadog-agent/pkg/util/winutil"
+
+	model "github.com/DataDog/agent-payload/process"
+	"github.com/DataDog/datadog-agent/pkg/process/config"
+)
+
+// CollectSystemInfo collects a set of system-level information that will not
+// change until a restart. This bit of information should be passed along with
+// the process messages.
+func CollectSystemInfo(cfg *config.AgentConfig) (*model.SystemInfo, error) {
+	hi, err := platform.GetArchInfo()
+	if err != nil {
+		return nil, err
+	}
+	cpuInfo, err := cpu.GetCpuInfo()
+	if err != nil {
+		return nil, err
+	}
+	fmt.Printf("cpu info %v\n", cpuInfo)
+	mi, err := winutil.VirtualMemory()
+	if err != nil {
+		return nil, err
+	}
+	physCount, _ := strconv.ParseInt(cpuInfo["cpu_pkgs"], 10, 64)
+	logicalCount, _ := strconv.ParseInt(cpuInfo["cpu_logical_processors"], 10, 64)
+	clockSpeed, _ := strconv.ParseInt(cpuInfo["mhz"], 10, 64)
+	l2Cache, _ := strconv.ParseInt(cpuInfo["cache_size_l2"], 10, 64)
+	cpus := make([]*model.CPUInfo, 0)
+	for i := int64(0); i < physCount; i++ {
+		cpus = append(cpus, &model.CPUInfo{
+			Number:     int32(i),
+			Vendor:     cpuInfo["vendor_id"],
+			Family:     cpuInfo["family"],
+			Model:      cpuInfo["model"],
+			PhysicalId: "",
+			CoreId:     "",
```

Comment:
Is it okay to lose these fields?

Reply:
those weren't being filled in by the underlying `gopsutil` anyway